packages/web-components/src/stories/container.stories.mdx

```diff
@@ -0,0 +1,243 @@
+import { extractArgTypes } from '@astrouxds/storybook-addon-docs-stencil'
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import { html, render } from 'lit-html'
+
+<Meta
+    title="Components/Container"
+    component="rux-container"
+    argTypes={extractArgTypes('rux-container')}
+/>
+
+# Container
+
+By default, Container's have a default padding of 1.25rem (20px) set on the body. If you'd like to change this (for example, using tables or a list inside your Container), you can use the `--body-padding` CSS custom property for convenience or the `body` CSS shadow part.
```

Comment:
Probably don't need 2 defaults here. By default, Containers have a padding of ....